templates/home.html

```diff
@@ -1,17 +1,13 @@
 {{ define "content" }}
+  {{ include "partials/flashes" }}
+  
+  <h1 class="sm:text-5xl md:text-6xl">
+    <span class="text-indigo-600">Go</span>ogle Scraper
+  </h1>
 
-<h1 class="text-primary">HELLO GOLANG</h1>
-{{ include "partials/flashes" }}
-
-{{ if .data.authenticatedUser }}
-  <h2 class="greeting">Welcome {{ .data.email }}</h2>
-  <h2><a href="/keyword">Keyword</a></h2>
-  <form class="mt-8 space-y-6" action="/logout" method="POST">
-    <button type="submit">Logout</button>
-  </form>
-{{ else }}
-  <h2><a href="/login">Login</a></h2>
-  <h2><a href="/register">Register</a></h2>
-{{ end }}
+  <h4 class="display-subtitle sm:mt-5 md:text-xl">
```

Comment:
These classes should be moved to the stylesheets instead (same issue above as well). Not sure why micing inline Tailwid classes with `display-subtitle`.

Reply:
Because Tailwind's `@apply` doesn't support non utility classes such as media-query.
But I found a work around. We can do something like this
```css
h1 {
  @apply .text-size-3;
  @screen lg {
    @apply .text-size-5;
  }
}

Should we move the inline media-query class to the stylesheets as above example?
```

Reply:
Yes, that's actually how we do it with Bootstrap too:=. Here is an example below from Compass's codebase:

https://github.com/nimblehq/compass/blob/83d644d3c6a91af2c0537216dbdab8f109a29b3e/_sass/screens/_team-developer-track.scss#L17-L27

Reply:
@gutakk yes, using `@screen` directive is a good solution. It would make the code clearer

Reply:
Fixed in a6e038f935ccaff5800ed8f2e6ddcfd998182677